plugins/atlas/skills/servicex/assets/xaod_hints.py

```
@@ -0,0 +1,131 @@
+
+    # Build a function type-shed that tells `func_adl` what the function signature is.
+    # This is used to generate the correct C++ code for the function.
+    def tool_call(**arg_dict):
+        """
+        NOTE: This is a dummy function that injects C++ into the object stream to do the
+        actual work.
+        """
+        ...
+
+    tool_call.__name__ = function_name
+    tool_call.__annotations__["return"] = eval(return_type_python)
```

Ruff (0.15.10)

[error] Use of possibly insecure function; consider using `ast.literal_eval`

(S307)